modules/op-results-db.adoc

```diff
@@ -4,37 +4,45 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="results-db_{context}"]
-= Preparing a secret with the database credentials
+= Configuring an external database server
 
-{tekton-results} uses a PostgreSQL database to store data. You can configure the installation to use either a PostgreSQL server that is automatically installed with {tekton-results} or an external PostgreSQL server that already exists in your deployment. In both cases, provide a secret with the database credentials.
+{tekton-results} uses a PostgreSQL database to store data. By default, the installation includes an internal PostgreSQL instance.
+You can configure the installation to use an external PostgreSQL server that already exists in your deployment.
 
 .Procedure
 
-Complete one of the following steps:
-
-* If you do not need to use an external PostgreSQL server, create a secret with the database user named `result` and a random password in the `openshift-pipelines` namespace by entering the following command:
+. Create a secret with the credentials for connecting to your PostgreSQL server by entering the following command:
 +
 [source,terminal]
 ----
 $ oc create secret generic tekton-results-postgres \
   --namespace=openshift-pipelines \
-  --from-literal=POSTGRES_USER=result \
-  --from-literal=POSTGRES_PASSWORD=$(openssl rand -base64 20)
+  --from-literal=POSTGRES_USER=<user> \ # <1>
+  --from-literal=POSTGRES_PASSWORD=<password> #<2>
```

Comment:
`#1` and `#2` are not explained anywhere. IMHO they are self-explanatory so maybe just remove those number references.

Reply:
fixed the formatting of the explanation

Reply:
removed the explanation entirely as it broke the build